events/models.py

```diff
@@ -0,0 +1,26 @@
+from django.db import models
+from django.utils import timezone
+
+from users.models import User
+from organization.models import InternalGroupPosition
+
+
+class Event(models.Model):
+    class Meta:
+        default_related_name = "events"
+        verbose_name_plural = "Events"
+
+    title = models.CharField(unique=True, max_length=80)
+    description = models.CharField(max_length=1024)
+    date = models.DateField(blank=True, null=True)
+
+    max_participants = models.IntegerField(default=10)
+    participants = models.ManyToManyField(User, blank=True)
```

Comment:
Think we might make the reverse relation a bit more clearer with a more explicit `related_name`, maybe something like this?
```suggestion
    participants = models.ManyToManyField(User, blank=True), related_name="attended_events"
```